src/frontend/next/src/pages/_app.tsx

```diff
@@ -1,8 +1,17 @@
+import { useEffect } from 'react';
 import { AppProps } from 'next/app';
 import '../styles/globals.css';
 
-const App = ({ Component, pageProps }: AppProps) => {
+const AppComponent = ({ Component, pageProps }: AppProps) => {
+  // This hook is for ensuring the styling is sync between client and server
```

Comment:
Let's add a link back to where this code came from, namely https://github.com/mui-org/material-ui/blob/master/examples/nextjs/pages/_app.js